### PR TITLE
feat: Wallet account view

### LINKED ISSUE #536: Wallet assets listing
### Please describe the problem that your request should resolve.
Ability to display Superhero Wallet assets

### Describe the solution you'd like
add wallet connection
add menu wallet (user) element
add subpage with assets listing and balances

### Is it already possible to achieve the same outcome in any other way? If so, how?
No

### PR DESCRIPTION
**Pull request by janmichek**
## Description
fixes #536 

There are 2 minor UX issues I will try to resolve in the following (Portfolio) issue, so I don't block the review

1. After successful wallet connection there is no loader when redirecting to connected account detail
2. When clicking on Try again, the whole page gets reloaded, which is not necessary. Only scanning again should work, but for some reason it's not working https://github.com/aeternity/aescan/issues/572

## Demo

https://github.com/aeternity/aescan/assets/15363559/64887a9e-5d48-4f4d-ad37-92793c2700ff

## Checklist:

- [x] I have read and followed the [Contributing Guide](../../CONTRIBUTING.md)  
- [x] add wallet account detail

### CODE REVIEW

File: src/components/TheHeader.vue [hunk 1]
```diff
@@ -36,6 +36,8 @@
         :class="[
           'header__network-select',
           { 'header__network-select--open': isNavigationOpen }]"/>
+
+      <the-wallet-account-controls class="u-hidden-mobile"/>
```
Comment (janmichek) [inline src/components/TheHeader.vue:40]: for now the feature is hidden on mobile as discussed

File: src/styles/utilities/_responsive.css [hunk 2]
```diff
@@ -5,8 +5,8 @@
 }
 
 .u-hidden-mobile {
-  display: none;
+  display: none !important;
```
Comment (janmichek) [inline src/styles/utilities/_responsive.css:8]: Here the usage of !importatnt is completely valid. 
Concept taken from bootstrap 
image

**Comment by Liubov-crypto** [comment]
@janmichek  I have questions: 

1. If I have more than 1 account in my wallet by switching them should the info also updating properly on scan page? Now I have to do it manually be refreshing the page and reconnecting the wallet:

https://github.com/aeternity/aescan/assets/69896204/c93d2f8a-adf4-403f-8c83-fddda9b8ad29

2. When I'm Exit wallet and then trying to connect again detecting the wallet is not working properly, it asked me to download the wallet however I have it installed already. I have to refresh page to see proper Connection button. Should it work in this  way? 

https://github.com/aeternity/aescan/assets/69896204/525a8680-811e-454a-91f3-e59696e2578b

**Comment by Liubov-crypto** [comment]
> @janmichek I have questions:
> 
> 1. If I have more than 1 account in my wallet by switching them should the info also updating properly on scan page? Now I have to do it manually be refreshing the page and reconnecting the wallet:
> 
>  2023-11-03.1.04.33.mov 
> 2. When I'm Exit wallet and then trying to connect again detecting the wallet is not working properly, it asked me to download the wallet however I have it installed already. I have to refresh page to see proper Connection button. Should it work in this  way?
> 
>  2023-11-03.1.44.27.mov 
> I got an issue in console: err

The first issue has been fixed. 

The second one is reproducible for me in FF and Chrome browsers. 

errrr

@janmichek please check it.

**Comment by janmichek** [comment]
> > @janmichek I have questions:
> > 
> > 1. If I have more than 1 account in my wallet by switching them should the info also updating properly on scan page? Now I have to do it manually be refreshing the page and reconnecting the wallet:
> > 
> > 2023-11-03.1.04.33.mov
> > 2. When I'm Exit wallet and then trying to connect again detecting the wallet is not working properly, it asked me to download the wallet however I have it installed already. I have to refresh page to see proper Connection button. Should it work in this  way?
> > 2023-11-03.1.44.27.mov
> > I got an issue in console: err
> 
> The first issue has been fixed.
> 
> The second one is reproducible for me in FF and Chrome browsers.
> 
> errrr
> 
> @janmichek please check it.

I fixed that. The solution relies on a dirty solution. There has to be some bug in SDK. I will escalate the fix for proper solution.

**Comment by janmichek** [comment]
Attaching the reproduction for the disconnection bug.
https://github.com/aeternity/aepp-boilerplate-vue/pull/27
THe solution now relies on reloading the page, which is not that bad, but not clean solution at all.

**Comment by janmichek** [comment]
I also changed the solution for generating identicons.
After discussion with SH Wallet team - they say the correct solution is to use the link `https://avatars.z52da5wt.xyz/${hash}` instead of using external library (blockies).
So I adjusted that, so the icon is the same as in app and wallet. We got rid of one dependency, which is a good thing

**Review by Liubov-crypto — CHANGES_REQUESTED** [review]
@janmichek Wallet avatars look broken in FF and Chrome: 
 
chrome
im
ff

https://github.com/aeternity/aescan/assets/69896204/67d606e6-18e3-4c4f-a5d9-23b7fa3fe6fa

The issue with connection after exit wallet has been fixed.

**Comment by janmichek** [comment]
> @janmichek Wallet avatars look broken in FF and Chrome:
> 
> chrome im ff
> 2023-11-09.5.51.28.mov
> 
> The issue with connection after exit wallet has been fixed.

Good catch! Fixed

**Review by Liubov-crypto — CHANGES_REQUESTED** [review]
The issue with avatars displaying has been fixed. 

1. I have a question: I have chain names in my wallet, when I'm connecting with ae scan does display avatars accordingly? Now I see it as gravatars that belongs to account address but not the chain name: 

https://github.com/aeternity/aescan/assets/69896204/a5ece224-a57e-4b4c-b530-1cc6cc5938cb

2. When I'm exit the wallet and then connecting again the top Connect wallet button isn't responding. A little bit confusing for the end user. Is this expected behaviour? 

https://github.com/aeternity/aescan/assets/69896204/82800fc4-39a1-474c-89f2-4dfa6af70acb

**Comment by janmichek** [comment]
> The issue with avatars displaying has been fixed.
> 
>     1. I have a question: I have chain names in my wallet, when I'm connecting with ae scan does display avatars accordingly? Now I see it as gravatars that belongs to account address but not the chain name:
> 
> 
> 2023-11-16.4.37.32.mov
> 
>     2. When I'm exit the wallet and then connecting again the top Connect wallet button isn't responding. A little bit confusing for the end user. Is this expected behaviour?
> 
> 
> 2023-11-16.4.46.03.mov

1.  Maybe you could spark more light about the status of avatars. From what I know SH wallet team plan to step out of using those avatars. Do you have any inside info? If not, we can plan to implement it separately.

2. Well the top button is not responding because you are already on the connection subpage. The confusing thing is the same style and wording for actual wallet connection and navigation to connection page. I don't have any creative solution now. I can think of it

**Pull request closed without merging**